Failed to determine Java version for profile checkstyle-tests

Seeing this in libraries-bom:

 Failed to determine Java version for profile checkstyle-tests @ com.google.cloud:google-cloud-shared-config:0.2.1, /home/elharo/.m2/repository/com/google/cloud/google-cloud-shared-config/0.2.1/google-cloud-shared-config-0.2.1.pom, line 406, column 14

I was able to reproduce the error. Checking further.

```
[INFO] --- maven-surefire-plugin:2.22.1:test (default-test) @ integration-tests ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.google.cloud.MaximumLinkageErrorsTest
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.673 s <<< FAILURE! - in com.google.cloud.MaximumLinkageErrorsTest
[ERROR] testMaximumLinkageErrors(com.google.cloud.MaximumLinkageErrorsTest)  Time elapsed: 0.645 s  <<< ERROR!
com.google.cloud.tools.opensource.dependencies.MavenRepositoryException: 
org.apache.maven.project.ProjectBuildingException: Some problems were encountered while processing the POMs:
[ERROR] Failed to determine Java version for profile checkstyle-tests @ com.google.cloud:google-cloud-shared-config:0.2.1, /usr/local/google/home/suztomo/.m2/repository/com/google/cloud/google-cloud-shared-config/0.2.1/google-cloud-shared-config-0.2.1.pom, line 406, column 14

```

--- a/pom.xml
+++ b/pom.xml
@@ -556,43 +556,5 @@
         </plugins>
       </build>
     </profile>
-    <profile>
-      <id>autovalue-java7</id>
-      <activation>
-        <jdk>1.7</jdk>
-      </activation>
-      <properties>
-        <autovalue.version>1.4</autovalue.version>
-      </properties>
-      <dependencyManagement>
-        <dependencies>
-          <dependency>
-            <groupId>com.google.auto.value</groupId>
-            <artifactId>auto-value</artifactId>
-            <version>${autovalue.version}</version>
-            <scope>provided</scope>
-          </dependency>
-        </dependencies>
-      </dependencyManagement>
-    </profile>
-    <profile>
-      <id>autovalue-java8</id>
-      <activation>
-        <jdk>[1.8,)</jdk>
-      </activation>
-      <properties>
-        <autovalue.version>1.6.6</autovalue.version>
-      </properties>
-      <dependencyManagement>
-        <dependencies>
-          <dependency>
-            <groupId>com.google.auto.value</groupId>
-            <artifactId>auto-value</artifactId>
-            <version>${autovalue.version}</version>
-            <scope>provided</scope>
-          </dependency>
-        </dependencies>
-      </dependencyManagement>
-    </profile>
   </profiles>
 </project>

